src/client.ts

```diff
@@ -15,14 +15,16 @@ import {
   describeCollection,
   deleteCollection,
 } from './control';
+import { Index } from './data';
 import { buildValidator } from './validator';
+import { queryParamsStringify } from './utils/queryParamsStringify';
 import { Static, Type } from '@sinclair/typebox';
 
 const ClientConfigurationSchema = Type.Object(
   {
     environment: Type.String({ minLength: 1 }),
     apiKey: Type.String({ minLength: 1 }),
-    projectId: Type.Optional(Type.String({ minLength: 1 })),
+    projectId: Type.String({ minLength: 1 }),
```

Comment:
Was it intentional to include these changes here? The README doesn't include projectId, so you should probably add it there at the same time